src/plugins/intel_gpu/tests/fusions/fusion_test_common.hpp

```diff
@@ -233,7 +233,11 @@ class BaseFusingTest : public ::testing::TestWithParam<T> {
     }
 
     layout get_activations_zp_layout(T& p) {
-        return layout{ p.data_type, p.default_format, tensor{1, p.in_shape.feature[0], 1, 1} };
+        auto shape = tensor{1, p.in_shape.feature[0], 1, 1};
+        // onednn doesn't support per-channel azp in convolution
+        if (engine.get_device_info().supports_immad)
+            shape.feature[0] = 1;
+        return layout{p.data_type, p.default_format, shape};
```

Comment:
This issue might be resolved with onednn v3.0 integration: https://github.com/openvinotoolkit/openvino/pull/14647